internal/pkg/flink/internal/store/store.go

```diff
@@ -157,6 +157,13 @@ func (s *Store) waitForPendingStatement(ctx context.Context, statementName strin
 			return nil, &types.StatementError{Msg: "Result retrieval aborted. Statement will be deleted.", HttpResponseCode: 499}
 		default:
 			statementObj, err := s.client.GetStatement(s.appOptions.GetOrgResourceId(), s.appOptions.GetEnvId(), statementName)
+
+			// print status detail message if available
+			status := statementObj.GetStatus()
```

Comment:
This was added as debug-only block to know the current status. We don't want to show it from the retry loop till we think we're done.

Reply:
Got it, will remove!